examples/tfengine/generated/team/project_iam/terraform.tfvars

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+project_iam_members = {
+
+  0 = {
+    parent_ids = ["example-prod-data"]
+    bindings = {
+      "roles/browser" = [
+        "serviceAccount:runner@example-prod-apps.iam.gserviceaccount.com",
+      ],
+    }
+  },
+}
+
+storage_bucket_iam_members = {
```

Comment:
can we make these *_iam_members a list so we don't have the index as an explicit key. So it would look like

```
storage_bucket_iam_members = [
  {
    parent_ids = ["foo"],
    bindigs = {...}
  },
  {
    parent_ids = ["bar"],
    ...
  }
]
```

Reply:
the for_each meta-argument only accepts a map or a set of strings. list of objects is not accepted.

Reply:
Have you tried `toset` function to convert the list to a set?

There is example of using `for_each` with a list of object here. Maybe can do something similar.
- https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/variables.tf#L38
- https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/iam.tf
and
- https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/variables.tf#L61
- https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/main.tf

Reply:
let me know if you want further changes for this

Reply:
yes. Having the index at the front is not good

Reply:
Give it a try like the examples I mentioned above.